Sources/ArcGISToolkit/Components/Offline/Preplanned/PreplannedListItemView.swift

```diff
@@ -166,4 +167,12 @@ private extension LocalizedStringResource {
             comment: "The status text when a map area packaging failed."
         )
     }
+    
+    static var cancelled: Self {
```

Comment:
I'll move this extension to `LocalizedStringResource` as it is used by both models, if we agree to add the `.downloadCancelled` case.